### PR TITLE
Add checklist + replace_me title to issue-template

### PR DESCRIPTION
**Pull request by hackerncoder**

### CODE REVIEW

**Comment by TheFrenchGhosty** [comment]
@hackerncoder why did you scrape this PR exactly? It's a good idea.

**Comment by hackerncoder** [comment]
I wasn't working on it. And wanted to show so.

**Comment by TheFrenchGhosty** [comment]
@hackerncoder I'll finish it later.

**Comment by hackerncoder** [comment]
I think we could merge just this?

**Comment by TheFrenchGhosty** [comment]
@hackerncoder No, as I said before I'm kinda busy IRL, I'll try to do it soon, it can wait anyway

**Comment by hackerncoder** [comment]
What more than this do you think needs to be done?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 
 ---
 
-<!-- Please use the search function to check if the bug you found has already been reported by someone else -->
+<!-- The comments between these brackets won't show up in the issue. -->
 <!-- If you want to suggest a new feature please use "Feature request" instead -->
 <!-- If you want to suggest an enhancement to an existing feature please use "Enhancement" instead -->
 
+### Checklist 
+<!-- This checklist is COMPULSORY. The first box has been checked so you you can see how it is done. -->
```
[the remark this answers is not in the document]
Comment (hackerncoder) [reply .github/ISSUE_TEMPLATE/bug_report.md:15]: Yes. Fixed

**Comment by TheFrenchGhosty** [comment]
bump

**Comment by hackerncoder** [comment]
Stale and leaving.

**Pull request closed without merging**